tasks/info.ts

```diff
@@ -4,12 +4,9 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 task("info", "Fetch reserves and pair balances")
```

Comment:
This task is using TokenLib,either delete the task or replace with real implementation

Reply:
removed